Modules/Filtering/ImageFusion/include/itkLabelOverlayFunctor.h

```diff
@@ -88,18 +88,13 @@ class LabelOverlayFunctor
   }
 
   bool
-  operator!=(const LabelOverlayFunctor & l) const
+  operator==(const LabelOverlayFunctor & other) const
   {
-    bool areDifferent = Math::NotExactlyEquals(l.m_Opacity, m_Opacity) || l.m_BackgroundValue != m_BackgroundValue ||
-                        l.m_RGBFunctor != m_RGBFunctor;
-    return areDifferent;
+    return Math::ExactlyEquals(m_Opacity, other.m_Opacity) && m_BackgroundValue == other.m_BackgroundValue &&
+           m_RGBFunctor == other.m_RGBFunctor;
```

Comment:
@Leengit This modification shows that De Morgan is still alive , right?  :-D

Reply:
`!(De Morgan's laws are not fabulous || De Morgan's laws are not very useful)`